src/main/python/config/configuration.py

```diff
@@ -0,0 +1,9 @@
+from configparser import ConfigParser
+
+# Look, this isn't the best but I'm going for speed right now rather than enterprise integrity
+DEVELOPMENT_CONFIG_FILE_PATH = "/home/george/htbdev.cfg"
```

Comment:
I feel ya on that comment, but I'd just stick that file in the repo top-level and `.gitignore` it, so you can relpath it with `./`.

IRL, I'd give this a default file name (like `htbdev.cfg`, like you have), but also construct a `dict` of others to switch on based on an env var, for a var named just `CONFIG_FILE_PATH` or something.

Reply:
IRL, I'd have already made a microservice + API for config fetching or shoved it in a Vault.

Reply:
For this simple project, I'm fine just making it toplevel